### PR TITLE
[python] Add directly install without clone source code

### PR DESCRIPTION
**Pull request by zhongjiajie**

### CODE REVIEW

**Comment by zhongjiajie** [comment]
PTAL @jieguangzhou

**Review by jieguangzhou — DISMISSED** [review]
LGTM

File: dolphinscheduler-python/pydolphinscheduler/docs/source/start.rst [hunk 1]
```diff
@@ -86,6 +86,14 @@ which we hold in GitHub
 After you installed *PyDolphinScheduler*, please remember `start Python Gateway Service`_
 which waiting for *PyDolphinScheduler*'s workflow definition require.
 
+Above command will clone whole dolphinscheduler source code to local, maybe you want to install latest pydolphinscheduler
+package directly and do not case about other code(including Python gateway service code), you can execute command
```
Comment (kezhenxu94) [inline dolphinscheduler-python/pydolphinscheduler/docs/source/start.rst]:
```suggestion
package directly and do not care about other code(including Python gateway service code), you can execute command
```

File: dolphinscheduler-python/pydolphinscheduler/docs/source/start.rst [hunk 2]
```diff
@@ -86,6 +86,14 @@ which we hold in GitHub
...
+
+.. code-block:: bash
+
+   # Must escape the '&' character by adding '\' 
+   pip install -e git+https://github.com/apache/dolphinscheduler.git#egg=apache-dolphinscheduler\&subdirectory=dolphinscheduler-python/pydolphinscheduler
```
Comment (kezhenxu94) [inline dolphinscheduler-python/pydolphinscheduler/docs/source/start.rst]: You can avoid escaping by quoting the url

```suggestion
   pip install -e "git+https://github.com/apache/dolphinscheduler.git#egg=apache-dolphinscheduler&subdirectory=dolphinscheduler-python/pydolphinscheduler"
```

**Review by kezhenxu94 — DISMISSED** [review]
Looks good

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**